packages/open-next/src/adapters/response.ts

```diff
@@ -11,24 +11,24 @@ const BODY = Symbol();
 const HEADERS = Symbol();
 
 function getString(data) {
-  if (Buffer.isBuffer(data)) {
+  if (Buffer.isBuffer(data) || ArrayBuffer.isView(data)) {
```

Comment:
Should ArrayBuffer be converted like this:
```ts
Buffer.from(data).toString("utf8");
```

Reply:
Hmm when I do that, the result is empty.